modulemd/include/modulemd-2.0/modulemd-compression.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file is part of libmodulemd
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Fedora-License-Identifier: MIT
+ * SPDX-2.0-License-Identifier: MIT
+ * SPDX-3.0-License-Identifier: MIT
+ *
+ * This program is free software.
+ * For more information on the license, see COPYING.
+ * For more information on free software, see <https://www.gnu.org/philosophy/free-sw.en.html>.
+ */
+
+#pragma once
+
+#include <glib.h>
+
+G_BEGIN_DECLS
+
+/**
+ * SECTION: modulemd-compression
+ * @title: Modulemd Compression Helpers
+ * @stability: stable
+ * @short_description: Utility functions for working with compressed files.
+ */
+
+
+/**
+ * ModulemdCompressionTypeEnum:
+ * @MODULEMD_COMPRESSION_TYPE_DETECTION_FAILED: Autodetection failure
+ * @MODULEMD_COMPRESSION_TYPE_UNKNOWN_COMPRESSION: Unknown compression
+ * @MODULEMD_COMPRESSION_TYPE_NO_COMPRESSION: No compression
+ * @MODULEMD_COMPRESSION_TYPE_GZ_COMPRESSION: gzip compression
+ * @MODULEMD_COMPRESSION_TYPE_BZ2_COMPRESSION: bzip2 compression
+ * @MODULEMD_COMPRESSION_TYPE_XZ_COMPRESSION: LZMA compression
+ * @MODULEMD_COMPRESSION_TYPE_ZCK_COMPRESSION: zchunk compression
+ * @MODULEMD_COMPRESSION_TYPE_SENTINEL: Enum list terminator
+ * Since: 2.8
```

Comment:
Please add a blank line before the "Since: 2.8" so it doesn't get concatenated to the last enum's description.

Reply:
Fixed